src/declarations/month.ts

```diff
@@ -1,15 +1,26 @@
-import { ITransaction } from './transaction';
+import { ITransaction, TransactionType } from './transaction';
 
 export interface IBalance {
   'company_id': number;
   'date': string;
   'money': number;
 }
 
+export interface ITransactionResponse {
+  id: number;
+  description: string;
+  money: number;
+  date: string;
+  companyId: number;
+  recurringId?: number;
+  type: TransactionType;
+  notes?: string;
+}
+
 export interface IMonth {
```

Comment:
I don't know if it would be too much of an inconvenience, but if this is mainly used in the context of the API response, maybe renaming to IMonthResponse is more congruent with the ITransaction/ITransactionResponse types? I won't require the change. Just a thought.